wcivf/apps/core/models.py

```diff
@@ -18,7 +18,7 @@ class LoggedPostcode(TimeStampedModel):
     def __str__(self):
         return "{0}".format(self.postcode)
 
-def log_postcode(log_dict, blocking=True):
+def log_postcode(log_dict, blocking=False):
```

Comment:
`Can you explain this commit? ` :) 

I think that's the way it is on Master: https://github.com/DemocracyClub/WhoCanIVoteFor/blob/master/wcivf/apps/core/models.py#L21?

Instead of installing Redis, I changed that default.
The original commit was a mistaken/subconscious `-a` flag to `git commit` - certainly I'm not proposing that you change from what's on master as part of this change, and I have no opinions on the matter either way, I just don't want to sysadmin a Redis :)